### ISSUE TITLE
FR: clear/white view

### ISSUE THREAD

**Issue by robmarkcole** [ESA-PhiLab/iris#42]
This would allow displaying the annotations on a clear or white background, which I think may be easier to visualise than when overlaid on an image. Not currently an option

<img width="241" alt="image" src="https://github.com/ESA-PhiLab/iris/assets/11855322/84389558-6251-4df2-9d6d-3da15355de5a">

**Comment by robmarkcole** [comment]
Previewing `mask.png` shows what I am suggesting:

<img width="414" alt="image" src="https://github.com/ESA-PhiLab/iris/assets/11855322/438518e4-3aed-45d6-a284-f9e673109e80">

**Comment by aliFrancis** [comment]
Absolutely! I think this is a nice idea. I will look into this in a future update. Perhaps it might be nice to be able to access something like $MASK in the config file, to allow for manipulation of the mask colours in new views, the default being this kind of clear-white/black background one.
